docs/requirements.in

```diff
@@ -0,0 +1,4 @@
+sphinx-autobuild>=0.7.1,<1.0
+sphinx
```

Comment:
Since Sphinx is not part of the public API, I'd set a minimum version of 3 or even 4. Just to have less things to potentially deal with.